pkg/agent/route/route.go

```diff
@@ -310,10 +322,120 @@ func (c *Client) RemoveServiceRouting() error {
 	ipRule := netlink.NewRule()
 	ipRule.IifName = c.nodeConfig.GatewayConfig.Name
 	ipRule.Mark = iptables.RtTblSelectorValue
+	ipRule.Mask = iptables.RtTblSelectorValue
 	ipRule.Table = AntreaServiceTableIdx
 	ipRule.Priority = AntreaIPRulePriority
 	if err = netlink.RuleDel(ipRule); err != nil {
 		return fmt.Errorf("ip rule delete: %w", err)
 	}
 	return nil
 }
+
+// resolveDefaultRouteNHMAC resolves the MAC of default route next
+// hop on service route table.
+func (c *client) resolveDefaultRouteNHMAC() (net.HardwareAddr, error) {
+	// This MAC is relevant to L2 mode only.
+	// TODO, for now uses AKS default MAC.
+	return net.ParseMAC("12:34:56:78:9a:bc")
+}
+
+// SetupPolicyOnlyMode configures routing needed by traffic policy-only mode.
+func (c *client) SetupPolicyOnlyMode() error {
+	gwLink := util.GetNetLink(c.nodeConfig.GatewayConfig.Name)
+	_, gwIP, _ := net.ParseCIDR(fmt.Sprintf("%s/32", c.nodeConfig.NodeIPAddr.IP.String()))
+	if err := netlink.AddrReplace(gwLink, &netlink.Addr{IPNet: gwIP}); err != nil {
+		klog.Fatalf("Failed to add address %s to gw %s: %v", gwIP, gwLink.Attrs().Name, err)
+	}
+
+	// add default route to service table.
+	_, defaultRt, _ := net.ParseCIDR("0/0")
+	nhIP := net.ParseIP("169.254.253.1")
```

Comment:
How this IP is decided? We just need a virtual IP not conflicts with others? If so, define constant.

Reply:
move to const